src/main/java/randoop/contract/CompareToSubs.java

```diff
@@ -0,0 +1,72 @@
+package randoop.contract;
+
+import randoop.Globals;
+
+/**
+ * The contract: Checks the substitutability property of compare to for equals
```

Comment:
Change "compare to" to compareTo; the former is confusing because it doesn't clearly reference a method name and on first reading I didn't realize it meant exactly that method.